build/lib/src/builder/build_step_impl.dart

```diff
@@ -110,18 +121,15 @@ class BuildStepImpl implements ManagedBuildStep {
     return done;
   }
 
-  @override
-  Future<Resolver> resolve(AssetId id,
-      {bool resolveAllConstants, List<AssetId> entryPoints}) async {
-    entryPoints ??= [];
-    if (!entryPoints.contains(id)) entryPoints.add(id);
-    return _resolvers.get(this, entryPoints, resolveAllConstants);
-  }
-
   /// Should be called after `build` has completed. This will wait until for
-  /// [_outputsCompleted].
+  /// [_outputsCompleted] and release all [Resolver]s that were created.
```

Comment:
[nit] Doc comment could use a title sentence

/// Wait for work to finish and clean up resources.
///
/// This method should be called after `build` has completed. After this Future resolves all outputs have been written and the [Resolver] for this build step - if any - has been released.